CHANGELOG.md

```diff
@@ -17,7 +17,7 @@
 -----------
 
 ### Internals
-* None.
+* Fix client reset failure during sync migration due to previous incomplete client reset. ([PR #7542](https://github.com/realm/realm-core/pull/7542), since v13.11.0)
```

Comment:
I think this actually belongs under the Fixed section rather than the Internals section.

Reply:
Also would be helpful to describe how the client reset failure would manifest. Like what would the error message be for users?

Reply:
Sure